backend/src/main/java/wooteco/prolog/studylog/exception/StudylogReadNotExistException.java

```diff
@@ -0,0 +1,8 @@
+package wooteco.prolog.studylog.exception;
+
+import wooteco.prolog.common.exception.BadRequestException;
+
+public class StudylogReadNotExistException extends
+        BadRequestException {
```

Comment:
```suggestion
public class StudylogReadNotExistException extends BadRequestException {
```

여기 개행 잘못 들어간 것 같아요 ~!

Reply:
오~~~ 짱 빠른데~~

Reply:
사실 저도 이 개행이 완전 맘에 안들지만 Scrap 을 구현하신 @sihyung92 님께서
Exception을 모두 이런 개행으로 짜셨길래 일단 맞춰서 써봤습니다...

![image](https://user-images.githubusercontent.com/56033755/151700049-abc896a2-904f-44af-91dc-976eac500821.png)

Reply:
개행 전부 수정했습니다 ~~~~~! 😆

Reply:
오호 성시형씨 개행 스타일이 조금 특이하네요 ~~

Reply:
역시 평범한 건 거부하는 ESFP~~